google_project/main.tf

```diff
@@ -60,3 +60,11 @@ AND NOT protoPayload.serviceName="iam.googleapis.com"
 AND NOT protoPayload.serviceName="sts.googleapis.com"
   EOT
 }
+
+resource "google_logging_project_bucket_config" "project" {
+  count            = var.log_analytics ? 1 : 0
```

Comment:
Per the previous comment, let's remove this so we define this resource for all projects.